katas/metal-slug/kevin/CharacterStatus.test.js

```diff
@@ -0,0 +1,9 @@
+import { CharacterStatus } from './CharacterStatus.js';
+test('Verify that getLifePoints get the life points of a soldier in a Tank', () => {
+    const setcharacterStatus = new CharacterStatus();
```

Comment:
Try to avoid verbs as the first word in a variable name that contains objects/values, this could create confusion with function names.